PYIC-7832: Update journey map with new page

AmritSidhu opened the pull request:
<!-- Provide a general summary of your changes in the Title above -->
<!-- Include the Jira ticket number in square brackets as prefix, eg `[P4-XXXX] PR Title` -->

## Proposed changes

### What changed

pyi-cri-escape and pyi-suggest-other-options templates will now be combined into a single dynamic template with URL 

photo-id-security-questions-find-another-way This also remove the no-F2F versions of this page and route to error in the case F2F is disabled

### Why did it change

<!-- Describe the reason these changes were made - the "why" -->

### Issue tracking
<!-- List any related Jira tickets or GitHub issues -->
<!-- List any related ADRs or RFCs -->
<!-- Delete/copy as appropriate -->

- [PYIC-7832](https://govukverify.atlassian.net/browse/PYIC-7832)

## Checklists

### Environment variables or secrets

<!-- Delete if changes DO include new environment variables or secrets -->
- [x] No environment variables or secrets were added or changed


[PYIC-7832]: https://govukverify.atlassian.net/browse/PYIC-7832?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

AmritSidhu committed:
PYIC-7832: Update journey map with new page

pyi-cri-escape and pyi-suggest-other-options templates will now be combined into a single dynamic template with URL photo-id-security-questions-find-another-way
This also remove the no-F2F versions of this page and route to error in the case F2F is disabled

AmritSidhu committed:
PYIC-7832: Add missing targetJourney

blakeyp commented on lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml:
```diff
@@ -328,25 +328,10 @@ states:
         targetJourney: F2F_HAND_OFF
         targetState: START
-
-  PYI_CRI_ESCAPE_NO_F2F:
```
For P2 we should probably keep the old states initially (and remove in a follow-up PR) to stage this out in case there any in flight journeys in these states as the deploy goes through

blakeyp commented on lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml:
```diff
@@ -492,26 +464,13 @@ states:
-            auditEvents:
-              - IPV_MITIGATION_START
-            auditContext:
-              mitigationType: enhanced-verification
```
It's probably not explicit in the ticket sorry, but as we're doing this could we also handle `f2f` disabled in `PYI_KBV_DROPOUT_NO_PHOTO_ID` and `MITIGATION_KBV_FAIL_NO_PHOTO_ID` same as photo id?

blakeyp commented on lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml:
```diff
@@ -358,31 +343,18 @@ states:
         targetState: START
       enhanced-verification:
-        targetState: MITIGATION_02_OPTIONS_WITH_F2F
+        targetState: PYI_KBV_DROPOUT_PHOTO_ID
```
Hmm don't we still need a separate state for this, as the page will be context dependent - like `PYI_KBV_DROPOUT_NO_PHOTO_ID` and `MITIGATION_KBV_FAIL_NO_PHOTO_ID` for no photo id?

AmritSidhu committed:
PYIC-7832: Add back original states + fail routing and context

Keep original states for now so it does not break any exisiting journeys, will be removed in follow up PR
We need to version of the new route, one with the context dropout
